testsuite/integration-arquillian/servers/auth-server/jboss/common/jboss-cli/keycloak-server-subsystem.cli

```diff
@@ -12,6 +12,10 @@ echo ** Adding login-protocol spi **
 /subsystem=keycloak-server/spi=login-protocol/:add
 /subsystem=keycloak-server/spi=login-protocol/provider=saml/:add(enabled=true,properties={knownProtocols => "[\"http=${auth.server.http.port}\",\"https=${auth.server.https.port}\"]"})
 
+echo ** Adding vault spi **
+/subsystem=keycloak-server/spi=vault/:add
+/subsystem=keycloak-server/spi=vault/provider=plaintext/:add(enabled=true,properties={dir => "${jboss.home.dir}/standalone/configuration/vault"})
+
```

Comment:
I think we should update also default server configuration accordingly. 
- https://github.com/keycloak/keycloak/tree/master/wildfly/server-subsystem

And migration scripts should be updated as well:
- https://github.com/keycloak/keycloak/tree/master/distribution/feature-packs/server-feature-pack/src/main/resources/content/bin

Reply:
I don't think we should be adding anything to the server config by default because the plaintext provider needs to be configured properly with the location of the directory containing the secret files. And AFAIK there is no sensible default for this at the moment.

Not saying we can't do it, just that we need to discuss what would a reasonable default be before doing it. And I think this could be handled by a separate task in Jira.

Reply:
If we agree on default empty vault provider suggested above, we would remove this file from the PR, and only set up the vault in this way in `testsuite/integration-arquillian/servers/auth-server/jboss/common`.

Reply:
I like the idea, in that case (if we agree on this) the conversation can be resolved.

Reply:
I don't follow how we are able to remove this file simply by having a default provider. We still need to configure the plaintext provider for the KeycloakVaultTest in order to run it with -Pauth-server-wildfly, no?

Reply:
You're right, I was wrongly looking at this file as if that was for migration.